brian2/utils/caching.py

```diff
@@ -0,0 +1,92 @@
+'''
+Module to support caching of function results to memory (used to cache results
+of parsing, generation of state update code, etc.). Provides the `cached`
+decorator.
+'''
+
+import functools
+import collections
+
+
+class _CacheStatistics(object):
```

Comment:
Yep - fine to leave it there anyway. We might find a use for it one way or the other.